internal/cephfs/controllerserver.go

```diff
@@ -712,3 +712,27 @@ func (cs *ControllerServer) DeleteSnapshot(ctx context.Context, req *csi.DeleteS
 
 	return &csi.DeleteSnapshotResponse{}, nil
 }
+
+// checkSubvolumeHasFeature verifies if the referred subvolume has
+// the required feature.
+func checkSubvolumeHasFeature(ctx context.Context, feature string, volID volumeID, cr *util.Credentials, volOptions *volumeOptions) (bool, error) {
```

Comment:
Sorry I missed this one, This function should be defined in volume.go not in controllerserver.go